nifi-pulsar-processors/src/main/java/org/apache/nifi/processors/pulsar/AbstractPulsarConsumerProcessor.java

```diff
@@ -76,6 +70,11 @@
     static final AllowableValue FAIL = new AllowableValue(ConsumerCryptoFailureAction.FAIL.name(), "Fail",
             "Report a failure condition, and then route the message contents to the FAILED relationship.");
 
+    static final AllowableValue OFFSET_EARLIEST = new AllowableValue("Earliest", "Earliest",
+            "Set the offset to the earliest position");
+    static final AllowableValue OFFSET_LATEST = new AllowableValue("Latest", "Latest",
+            "Set the offset to the latest position");
```

Comment:
Same comment as above, Make this "The latest position which means the start consuming position will be the last message."